config/unleash.php

```diff
@@ -24,6 +24,6 @@
     'applicationHostname' => \MikeFrancis\LaravelUnleash\Strategies\ApplicationHostnameStrategy::class,
     'default' => \MikeFrancis\LaravelUnleash\Strategies\DefaultStrategy::class,
     'remoteAddress' => \MikeFrancis\LaravelUnleash\Strategies\RemoteAddressStrategy::class,
-    'userWithIds' => \MikeFrancis\LaravelUnleash\Strategies\UserWithIdStrategy::class,
+    'userWithId' => \MikeFrancis\LaravelUnleash\Strategies\UserWithIdStrategy::class,
```

Comment:
Why has this changed? Was it incorrect?

Reply:
This is the reponse of a feature with corresponding strategy which I get from unleash api/client/features.
`{`
`    "name": "user_ids",`
`    "description": "",`
`    "type": "release",`
`    "project": "default",`
`    "enabled": true,`
`    "stale": false,`
`    "strategies": [`
`        {`
`            "name": "userWithId",`
`            "parameters": {`
`                "userIds": "qing,bram"`
`            }`
`        }`
`    ],`
`    "variants": null,`
`    "createdAt": "2020-12-05T23:58:37.233Z"`
`}`
The key is userWithId without s.
What response do you get from the api? Maybe there is version difference?

Reply:
Ok, will take a deeper look at this. Thanks for fixing.